shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/uimodel/QuestionUiModel.kt

```diff
@@ -32,3 +34,14 @@ fun Question.toQuestionUiModel(index: Int, totalStep: Int): QuestionUiModel = Qu
     question = this,
     step = "${index + 1}/${totalStep}"
 )
+
+fun List<QuestionUiModel>.toQuestionSubmissions(): List<QuestionSubmission> {
+    return this.map { question ->
+        QuestionSubmission(
+            id = question.id,
+            answers = question.answers.map { answer ->
```

Comment:
@phongvhd93 could you please check if we should map `answers` or `userInputs` here? 🤔

Reply:
solved in 8b0c043